src/Root.tsx

```diff
@@ -0,0 +1,26 @@
+import { Route, Routes, HashRouter, Navigate } from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/pages/HomePage';
+import { PeoplePage } from './components/pages/PeoplePage';
+import { NotFoundPage } from './components/pages/NotFoundPage';
+import { FC } from 'react';
+
+export const Root: FC = () => {
+  return (
+    <HashRouter>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+          <Route path="home" element={<Navigate to="/" replace />} />
```

Comment:
It's not common to redirect from '/home' to '/'. If both routes are supposed to show the same content, it's better to render the same component for both of them. For example, you could have both `<Route index element={<HomePage />} />` and `<Route path='home' element={<HomePage />} />`.